release/edgex-go.yaml

```diff
@@ -1,11 +1,11 @@
 ---
 name: 'edgex-go'
-version: '4.0.1'
+version: '4.0.0'
```

Comment:
The edgex-go repo has both v4.0.0 and v4.0.1 tags, is it ok to specify 4.0.0 here?